src/main/java/teammates/storage/entity/Instructor.java

```diff
@@ -18,6 +19,15 @@
 public class Instructor extends BaseEntity {
 
     /**
+    * Sort the Instructors list alphabetically by name.
+    */
+    public static Comparator<Instructor> compareByName = new Comparator<Instructor>() {
+        public int compare(Instructor one, Instructor other) {
```

Comment:
This should be moved to `InstructorAttributes`, which should implement `Comparable` (use `FeedbackQuestionAttributes` as an example).